src/Runner.Worker/Container/ContainerHooks/HookContainer.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace GitHub.Runner.Worker.Container.ContainerHooks
+{
+    public class HookContainer : HookArgs
+    {
+        public string Id { get; set; }
+        public string DisplayName { get; set; }
+        public string Network { get; set; }
```

Comment:
Are network or network alias needed? Are they sent over?

Reply:
I've put these fields (network, id, ports) into a class called `ResponseContainer`, to be read from `response.json` and used to populate the fields mentioned above in JobContext